lib/taskana-core-test/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java

```diff
@@ -394,4 +409,40 @@ void should_ForceCancelClaimTask_When_InReviewByAnotherUser() throws Exception {
     assertThat(unclaimedTask.isRead()).isTrue();
     assertThat(unclaimedTask.getOwner()).isNull();
   }
+
+  @Nested
+  @TestInstance(Lifecycle.PER_CLASS)
+  class WithAdditionalUserInfoEnabled implements TaskanaConfigurationModifier {
+
+    @TaskanaInject TaskService taskService;
+
+    @Override
+    public TaskanaConfiguration.Builder modify(TaskanaConfiguration.Builder builder) {
+      return builder.addAdditionalUserInfo(true);
+    }
+
+    @WithAccessId(user = "user-1-2")
+    @Test
+    void should_SetOwnerLongName() throws Exception {
```

Comment:
shoudl we add a "when" clause to the test name? like should_SetOwnerLongName_When_AdditionalUserInfoEnabled() or soemthing like that?

Reply:
Oh I didn't see that it was in that nested class...or rather Github doesnt show it in the lines changed...I think this is fine then the way it is